SU-SOE/soe_basic
# D8CORE-3386: fixing the mailto link in footer

# READY FOR REVIEW

# Summary
- Fixing the colors in the local footer for SOE with a mailto link.

# Review By (Date)
- 1/15

# Urgency
- highish

# Steps to Test

1. Pull in this change.
2. Add content with a mailto link to the local footer. /admin/config/system/local-footer
3. Verify the envelop matches color and the hover has an underline on the text and the icon changes color correctly. 

# Affected Projects or Products
- SoE Subtheme

# Associated Issues and/or People
- D8CORE-3386
- @rmundstock 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
